components/AdminFunc/UpdateClass.tsx

```diff
@@ -73,9 +81,9 @@ export default function UpdateClass({ id }) {
 
   const { inputs, handleChange } = useForm(data?.SportClass)
 
-  const onSubmit = async (e) => {
+  const onSubmit = async (e: React.FormEvent<HTMLFormElement>) => {
     e.preventDefault()
-    const res = await updateSportClass({
+    await updateSportClass({
```

Comment:
Why isn't this `async/await` wrapped with a `try/catch`?

Reply:
if there's an error from the mutation, the DisplayError component catches it and displays whether its a graphql or network error, thought that was enough.

Reply:
I still wrap mine and did catch a weird error that way, I also needed to do some extra work on the login and sig in mutations